src/navigator/Headers.tsx

```diff
@@ -192,7 +192,7 @@ export function HeaderTitleWithSubtitle({
   subTitle,
   testID,
 }: {
-  title: string | React.ReactNode
+  title?: string | React.ReactNode
```

Comment:
nothing to do but man, a HeaderTitleWithSubtitle component that does not require a title _or_ a subtitle 🙈